components/image/eog/patches/eog-07-xopensource.patch

```diff
@@ -0,0 +1,25 @@
+--- eog-2.32.1/src/eog-util.c.orig	Sun May  7 10:29:51 2017
++++ eog-2.32.1/src/eog-util.c	Sun May  7 10:30:38 2017
+@@ -28,7 +28,9 @@
+ 
+ #include <sys/time.h>
+ #ifdef HAVE_STRPTIME
++#ifndef _XOPEN_SOURCE
```

Comment:
Perhaps, remove it or #ifndef __sun ? Defining _XOPEN_SOURCE in code is usually harmful on OI